src/Grpc.Core.Api/ClientBase.cs

```diff
@@ -84,6 +86,9 @@ public T WithHost(string host)
 /// <summary>
 /// Base class for client-side stubs.
 /// </summary>
+// The call invoker's debug information is provided by DebuggerDisplayAttribute. It isn't available in ToString.
```

Comment:
```suggestion
// The call invoker's debug information is specified in DebuggerDisplayAttribute.
// It can't be concatenated inside ServiceNameDebuggerToString() because it isn't available in ToString.
```